config/default.yml

```diff
@@ -1,3 +1,7 @@
+AllCops:
+  Exclude:
+    - node_modules/**/*
```

Comment:
I guess this should only be present for Rails configs, because usual ruby projects don't have js dependencies